doc/administrator/upgrades/index.md

```diff
@@ -40,14 +41,93 @@ Note that if this process is not completed correctly for a Remote API, data will
+
+While investigating how these updates will be applied to IRIDA databases, we noticed some irregularities in the audit records for some existing metadata.  Because of this we're recommending additional backup procedures be taken before completing this upgrade.  
+
+To skip straight to the backup procedure, [click here](#metadata-linking-script) For a full technical description of the issue, keep reading below.
+
+### Database structure
+{:.no_toc}
+
+To fully understand this problem, we'll first describe the structure of our metadata system and relevant database tables.  Given the following example of a basic line-list:
+
+|         | Organism   |
+|---------|------------|
+| sample1 | Salmonella |
+
+The tables involved are the following:
+
+
+* `sample` - A record of a sample which contains its name, creation date, and some fixed metadata (ex: "sample1").
+* `metadata_field` - The header of a line-list column.  This table contains the field's name & datatype (ex: "Organism").
+* `metadata_entry` - The value of a cell in the line-list (ex: "Salmonella").
+* `sample_metadata_entry` - The relationship between the above 3 tables (ex: for "sample1", the "Organism" is "Salmonella").
+
+In addition to the above, there is a matching `_AUD` (audit) table for each of these tables which captures any changes to the data and the user & time which the change occurred.  This auditing is managed by [Hibernate Envers](https://hibernate.org/orm/envers/).
+
+In general data will be populated into these tables in the following order:
+
+1. `sample` entry is created as sequencing data is uploaded,
+2. Someone uploads a set of sample metadata, which will first get a `metadata_field`
+  a. Find an existing `metadata_field` for a column header,
+  b. Or create a new `metadata_field` for a header,
+3. `metadata_entry` is created with the appropriate value,
+4. `sample_metadata_entry` is created linking the above 3 items together.
+
+Note items 2, 3, and 4 will generally be done at the same time.  As soon as the metadata values are committed to the database, Envers should create `_AUD` entries for all of the above records.
+
+### The problem
+{:.no_toc}
+
+The issue we noticed is that while Envers seemed to create `_AUD` records for all the above tables when metadata is first added to a sample, subsequent updates of those values *may* not create records for the `sample_metadata_entry_AUD` table, though it properly creates records for the `metadata_entry_AUD` and `sample_AUD` tables.  We found this to be most prevalent when data is synchronized from remote IRIDA installations.  Most of these synchronized records were duplicate entries that were saving the same metadata values back to the sample, but creating new rows in the database.  We are still investigating why this issue was occurring.  Due to the complex nature of the above database tables it appears that Envers was failing to properly audit the relationship between those entities/tables.
```

Comment:
I'm not sure this is the exact issue, but it may be one of the cases where this appears.  Added info that re-running analysis pipelines may be part of the issue in 7baf075